test/shell/exec-usergroup.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+
+set -x
+
+if ! timeout 1s sudo -A true; then
+	echo "sudo disabled: not checking exec user and group"
+	exit
+fi
+
+function setup {
+	mkdir -p /tmp/mgmt/test-exec-usergroup/
+	sudo -A chown nobody:$group /tmp/mgmt/test-exec-usergroup/
+	sudo -A chmod 775 /tmp/mgmt/test-exec-usergroup/
+}
+
+function cleanup {
+	sudo -A rm -rf /tmp/mgmt/test-exec-usergroup
+}
+
+# run_test will run each test. It takes 3 parameters:
+# - $1: graph (e.g. exec-usergroup-nobody.yaml)
+# - $2: user to be tested (e.g. nobody or "")
+# - $3: group to be tested (e.g. nobody or "")
+function run_test() {
```

Comment:
please run this to something like run_usergroup_test as to avoid conflict with the similarly named run-test function in another bash file.